### PR TITLE
Ui

### PR DESCRIPTION
**Pull request by mozzy22**
this  PR includes all the commits . 
u can find the pages here https://mozzy22.github.io/Fast-Food-Fast/index.html

in order to view all the changes without going commit by comit , click on `files changed`

### CODE REVIEW

**Comment by katunold** [comment]
Hey, I have just seen an interface that looks exactly similar to this raised in a PR. Otherwise good job

**Comment by katunold** [comment]
Oooh, sorry. I think I had seen this. no similar copy as earlier said

File: UI/HTML/adminpage.html [hunk 1]
```diff
@@ -0,0 +1,342 @@
...
+
+
+      <div id="div3" class="tabcontent">
+          <table class = "orderTable">
+              <tr>
+                  <td> 
+                    <p class = "imagetitle">chicken pizza :20$ </p>
+                    <img class = "image" src= "../IMAGES/chickenPizza.jpg">
+                    <form>
+                      <dv>
```
Comment (66hakeem) [inline UI/HTML/adminpage.html]: This appears as <dv>, Did you mean to type <div>? Same occurs on more lines in this file.

Comment (mozzy22) [reply UI/HTML/adminpage.html]: oh its an error . thx  @66hakeem

**Comment by mozzy22** [comment]
thx @katunold  and @66hakeem  ,

**Comment by Luleherll** [comment]
I think you should make the food items bigger on mobile

**Comment by Ivankivu** [comment]
@mozzy22 I have visited your UI but the responsive adjustment is still having issues please clarify on that. Otherwise progress is fair

**Comment by mozzy22** [comment]
@Ivankivu  your ryt ... i ahdnt completed te responsiveness on the order page . thx

**Comment by mozzy22** [comment]
@Luleherll yourryt . thx

**Pull request closed without merging**